README.md

```diff
@@ -17,10 +17,10 @@ DataHelix is a proud member of the [Fintech Open Source Foundation](https://www.
 * For a high level road map see [Road Map](docs/RoadMap.md).
 
 ## The Problem
-When performing a wide range of software development tasks - functional or load testing on a system, prototyping an API or pitching a service to a potential customer - sample data is a necessity, but generating and maintaining it can be difficult. The nature of the financial services industry makes it particularly difficult to cleanly manage data schemas, and sample datasets:
+When performing a wide range of software development tasks - functional or load testing on a system, prototyping an API or pitching a service to a potential customer - sample data is a necessity, but generating and maintaining it can be difficult. The nature of the some industries makes it particularly difficult to cleanly manage data schemas, and sample datasets:
```

Comment:
>The nature of the some industries 
The nature of ~the~ some industries?

Also very nitpicky but we could use em dashes (—) rather than hyphens (-)?